<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>558</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QSplitter" name="splitterMain">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="topHorizontalSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTreeView" name="nodeTree">
        <property name="rootIsDecorated">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="animated">
         <bool>true</bool>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QTabWidget" name="tabWidget">
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>2</number>
        </property>
        <widget class="QWidget" name="workListTab">
         <attribute name="title">
          <string>Work Done</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableView" name="workList">
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>70</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="summaryTab">
         <attribute name="title">
          <string>Summary</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="summaryVerticalLayout">
            <item>
             <layout class="QHBoxLayout" name="summaryControlsLayout">
              <item>
               <widget class="QComboBox" name="summarySelectionCombo">
                <property name="currentText">
                 <string/>
                </property>
                <property name="currentIndex">
                 <number>-1</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="summarySelectionOptionsButton">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="toolTip">
                 <string>Select week</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="refreshSummaryButton">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/res/icons/refresh.svg</normaloff>:/res/icons/refresh.svg</iconset>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignBottom">
               <widget class="QLabel" name="summarySelectionLabel">
                <property name="text">
                 <string>unset</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="summaryFilterBtn">
                <property name="toolTip">
                 <string>Apply filter</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset>
                  <normalon>:/res/icons/filter.svg</normalon>
                 </iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>78</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="summaryView">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="showGrid">
               <bool>true</bool>
              </property>
              <property name="gridStyle">
               <enum>Qt::DotLine</enum>
              </property>
              <property name="cornerButtonEnabled">
               <bool>false</bool>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>60</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderHighlightSections">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="reportTab">
         <property name="accessibleName">
          <string/>
         </property>
         <attribute name="title">
          <string>Report</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="reportVerticalLayout">
            <item>
             <layout class="QHBoxLayout" name="reportControlsLayout">
              <item>
               <widget class="QPushButton" name="overviewFilter">
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="../resources.qrc">
                  <normaloff>:/res/icons/filter.svg</normaloff>:/res/icons/filter.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="overviewOptions">
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="reportList">
              <property name="cornerButtonEnabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
      <widget class="QSplitter" name="bottomHorizontalSplitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QTableView" name="currentWorkList">
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <layout class="QVBoxLayout" name="verticalWorkArea">
         <item>
          <widget class="QPushButton" name="startNewButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Start new</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="resumeButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Resume</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="doneButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Done</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="suspendButton">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Suspend</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Settings"/>
    <addaction name="action_Quit"/>
    <addaction name="action_About"/>
   </widget>
   <widget class="QMenu" name="menuNodes">
    <property name="title">
     <string>&amp;Nodes</string>
    </property>
    <addaction name="actionNodeTreeNew_Folder"/>
    <addaction name="actionNodeTreeNew_Customer"/>
    <addaction name="actionNodeTreeNew_Project"/>
    <addaction name="actionNodeTreeNew_Task"/>
    <addaction name="separator"/>
    <addaction name="actionTreeViewEdit_a_node"/>
    <addaction name="actionNodeTreeDelete_a_Node"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionscreen_size_1024x768"/>
    <addaction name="actionScreen_Size_1280x720"/>
    <addaction name="actionScreen_Size_1920x1080"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuNodes"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNodeTreeNew_Folder"/>
   <addaction name="actionNodeTreeNew_Customer"/>
   <addaction name="actionNodeTreeNew_Project"/>
   <addaction name="actionNodeTreeNew_Task"/>
   <addaction name="actionTreeViewEdit_a_node"/>
   <addaction name="actionNodeTreeDelete_a_Node"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Settings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionNodeTreeNew_Folder">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/new_folder.svg</normaloff>
     <normalon>:/res/icons/folder.svg</normalon>:/res/icons/new_folder.svg</iconset>
   </property>
   <property name="text">
    <string>New Folder</string>
   </property>
   <property name="toolTip">
    <string>Create a new Folder</string>
   </property>
  </action>
  <action name="actionNodeTreeNew_Customer">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/new_customer.svg</normaloff>
     <normalon>:/res/icons/customer.svg</normalon>:/res/icons/new_customer.svg</iconset>
   </property>
   <property name="text">
    <string>New Customer</string>
   </property>
   <property name="toolTip">
    <string>Create a new Customer</string>
   </property>
  </action>
  <action name="actionNodeTreeNew_Project">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/new_project.svg</normaloff>
     <normalon>:/res/icons/project.svg</normalon>:/res/icons/new_project.svg</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>Create a new Project</string>
   </property>
  </action>
  <action name="actionNodeTreeNew_Task">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/new_task.svg</normaloff>
     <normalon>:/res/icons/task.svg</normalon>:/res/icons/new_task.svg</iconset>
   </property>
   <property name="text">
    <string>New Task</string>
   </property>
   <property name="toolTip">
    <string>Create a new Task</string>
   </property>
  </action>
  <action name="actionFilter_and_Group_on_Summary">
   <property name="icon">
    <iconset>
     <normalon>:/res/icons/filter.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Filter and Group on Summary</string>
   </property>
   <property name="toolTip">
    <string>Apply rules for the summary</string>
   </property>
  </action>
  <action name="actionNodeTreeDelete_a_Node">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/delete.svg</normaloff>:/res/icons/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete a Node</string>
   </property>
   <property name="toolTip">
    <string>Delete a node</string>
   </property>
  </action>
  <action name="actionTreeViewEdit_a_node">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/edit.svg</normaloff>:/res/icons/edit.svg</iconset>
   </property>
   <property name="text">
    <string>Edit a node</string>
   </property>
   <property name="toolTip">
    <string>Edit a node</string>
   </property>
  </action>
  <action name="actionEdit_a_Current_Work_Item">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/edit.svg</normaloff>:/res/icons/edit.svg</iconset>
   </property>
   <property name="text">
    <string>Edit a Current Work Item</string>
   </property>
  </action>
  <action name="actionDelete_a_Current_Work_Item">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/delete.svg</normaloff>:/res/icons/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete a Current Work Item</string>
   </property>
  </action>
  <action name="actionWorkList_Edit_Item">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/edit.svg</normaloff>:/res/icons/edit.svg</iconset>
   </property>
   <property name="text">
    <string>Edit Item</string>
   </property>
   <property name="toolTip">
    <string>Edit Item</string>
   </property>
  </action>
  <action name="actionWorkList_Delete_Item">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/delete.svg</normaloff>:/res/icons/delete.svg</iconset>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
   <property name="toolTip">
    <string>Delete Item</string>
   </property>
  </action>
  <action name="actionWorkList_Set_as_Done">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/done.svg</normaloff>:/res/icons/done.svg</iconset>
   </property>
   <property name="text">
    <string>Set as Done</string>
   </property>
   <property name="toolTip">
    <string>Mark all selected items as done</string>
   </property>
  </action>
  <action name="actionWorkList_Set_as_Free">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/free.svg</normaloff>:/res/icons/free.svg</iconset>
   </property>
   <property name="text">
    <string>Set as Free</string>
   </property>
   <property name="toolTip">
    <string>Set as Free</string>
   </property>
  </action>
  <action name="actionWorkList_Set_as_Held_Back">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/heldback.svg</normaloff>:/res/icons/heldback.svg</iconset>
   </property>
   <property name="text">
    <string>Set as Held Back</string>
   </property>
   <property name="toolTip">
    <string>Set as Held Back</string>
   </property>
  </action>
  <action name="actionWorkList_Set_as_Approved">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/approved.svg</normaloff>:/res/icons/approved.svg</iconset>
   </property>
   <property name="text">
    <string>Set as Approved</string>
   </property>
   <property name="toolTip">
    <string>Set as Approved</string>
   </property>
  </action>
  <action name="actionWorkList_Set_as_Invoiced">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/res/icons/invoiced.svg</normaloff>:/res/icons/invoiced.svg</iconset>
   </property>
   <property name="text">
    <string>Set as Invoiced</string>
   </property>
   <property name="toolTip">
    <string>Set as Invoiced</string>
   </property>
  </action>
  <action name="actionscreen_size_1024x768">
   <property name="text">
    <string>Screen Size 1024x768</string>
   </property>
  </action>
  <action name="actionScreen_Size_1280x720">
   <property name="text">
    <string>Screen Size 1280x720</string>
   </property>
  </action>
  <action name="actionScreen_Size_1920x1080">
   <property name="text">
    <string>Screen Size 1920x1080</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>250</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>summaryFilterBtn</sender>
   <signal>clicked()</signal>
   <receiver>actionFilter_and_Group_on_Summary</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
